### ISSUE TITLE
Performance and functional comparison

### ISSUE THREAD

**Issue by big-andy-coates** [ssilverman/snowy-json#17]
Hi,

I've recently needed to compare the performance and functionality of this and other JVM based JSON validation libraries, and I thought you might like to see how this implementation compares.  I've published to the code and the results here: https://github.com/creek-service/json-schema-validation-comparison.

While I've spent time trying to get the best out of each library under test, it's possible that the results for this implementation could be improved if someone with more in-depth knowledge were to look into it.  If you feel I'm not showing in its best light, please feel free to raise PRs to fix issues and improve your score.

Please feel free to link to the code and results.

I hope this is of some use to you.

Thanks,

Andy

**Comment by big-andy-coates** [comment]
FYI, microsite with performance and functionality comparison is now live: https://www.creekservice.org/json-schema-validation-comparison/

**Comment by ssilverman** [comment]
@big-andy-coates thanks for compiling that data. Since I haven't worked on the project for more than 2.5 years, I'd expect there to be some errors and not 100% passing of all the tests. It passes most of them, and that makes me happy. :)

You're correct in your assessment that the reason the performance is poor compared to the other libraries was because I intended it as a reference implementation (and also a way for me to learn the subject). I spent zero time on optimization or performance tuning.
